lldb/source/Plugins/DynamicLoader/POSIX-DYLD/DynamicLoaderPOSIXDYLDProperties.td

```diff
@@ -0,0 +1,8 @@
+
+include "../../../../include/lldb/Core/PropertiesBase.td"
+
+let Definition = "dynamicloaderposixdyld" in {
+  def ParallelModuleLoad: Property<"parallel-module-load", "Boolean">,
```

Comment:
If we settle on keeping the per-plugin setting, this should be consistent with `plugin.dynamic-loader.darwin.experimental.enable-parallel-image-load`, i.e. be under experimental and use `enable-parallel-image-load` (even though I like your setting without the enable a little more). If we hoist it to target, then I'd like to go with `parallel-module-load`.

Reply:
I don't feel strongly about keeping it as a plugin-specific setting or hoisting it to target and setting it generically. I missed the naming for the Darwin-equivalent though -- will rename depending on what we choose. 

I initially opted for the plugin-scoped setting since I wanted to make the setting as specific as possible, because I wasn't sure how many other platforms had a specific parallel loading flag like Darwin.